views/package.json

```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.7.1",
-        "@oat-sa/tao-item-runner-qti": "0.21.11"
+        "@oat-sa/tao-item-runner-qti": "0.21.12"
```

Comment:
Shouldn't that be backported?

Reply:
It is, [0.21.12](https://github.com/oat-sa/tao-item-runner-qti-fe/releases/tag/v0.21.12) contains the backport of the fix.